src/EnrollChoicesController.js

```diff
@@ -103,6 +107,8 @@ function (Okta, FormController, Enums, RouterUtil, FactorList,
       },
 
       preRender: function () {
+        this.add(subtitleTpl({subtitle: this.getSubtitle()}), $('p.okta-form-subtitle'));
```

Comment:
mind adding a small comment about why we do this? Something like `// form subtitle does not support unescaped strings, and we need some html elements in the subtitle for this form. So instead of a regular subtitle, we add a <span> with the text we need`